velox/exec/CMakeLists.txt

```diff
@@ -11,56 +11,111 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(
-  velox_exec
-  Aggregate.cpp
-  AggregateFunctionRegistry.cpp
-  AggregationMasks.cpp
-  AggregateWindow.cpp
-  ContainerRowSerde.cpp
-  CrossJoinBuild.cpp
-  CrossJoinProbe.cpp
-  Driver.cpp
-  EnforceSingleRow.cpp
-  Exchange.cpp
-  FilterProject.cpp
-  GroupId.cpp
-  GroupingSet.cpp
-  HashAggregation.cpp
-  HashBuild.cpp
-  HashJoinBridge.cpp
-  HashPartitionFunction.cpp
-  HashProbe.cpp
-  HashTable.cpp
-  JoinBridge.cpp
-  Limit.cpp
-  LocalPartition.cpp
-  LocalPlanner.cpp
-  Merge.cpp
-  MergeJoin.cpp
-  MergeSource.cpp
-  Operator.cpp
-  OperatorUtils.cpp
-  OrderBy.cpp
-  PartitionedOutput.cpp
-  PartitionedOutputBufferManager.cpp
-  PlanNodeStats.cpp
-  RowContainer.cpp
-  Spill.cpp
-  SpillOperatorGroup.cpp
-  Spiller.cpp
-  StreamingAggregation.cpp
-  TableScan.cpp
-  TableWriter.cpp
-  Task.cpp
-  TopN.cpp
-  Unnest.cpp
-  Values.cpp
-  VectorHasher.cpp
-  Window.cpp
-  WindowFunction.cpp
-  WindowPartition.cpp
-  AssignUniqueId.cpp)
+
+if (VELOX_FOR_MILVUS STREQUAL "ON")
+  add_library(
+    velox_exec
+    Aggregate.cpp
+    AggregateFunctionRegistry.cpp
+    AggregationMasks.cpp
+    AggregateWindow.cpp
+    ContainerRowSerde.cpp
+    CrossJoinBuild.cpp
+    CrossJoinProbe.cpp
+    Driver.cpp
+    EnforceSingleRow.cpp
+    Exchange.cpp
+    FilterProject.cpp
+    FilterBitSet.cpp
```

Comment:
I think it's better to put the files that Milvus needs to add in a separate module called velox_milvus.